django/applications/catmaid/control/log.py

```diff
@@ -69,7 +71,7 @@ def list_logs(request, project_id=None):
     if should_sort:
         column_count = int(request.POST.get('iSortingCols', 0))
         sorting_directions = [request.POST.get('sSortDir_%d' % d, 'DESC') for d in range(column_count)]
-        sorting_directions = map(lambda d: '-' if d.upper() == 'DESC' else '', sorting_directions)
+        sorting_directions = list(map(lambda d: '-' if d.upper() == 'DESC' else '', sorting_directions))
```

Comment:
A Python-2 ism

Reply:
This shouldn't be a problem though. The iterator returned by `map` (in Python 3) can be consumed by `zip()`, which is where `sorting_directions` is used a few lines further down (like `sorting_cols` is used too). This endpoint also seems to work fine when sorting the log table in the front-end. Therefore, I don't think this change is needed.